frontend/src/lib/lemon-ui/LemonButton/LemonButton.tsx

```diff
@@ -63,7 +63,7 @@ export interface LemonButtonPropsBase
     /** Like plain `disabled`, except we enforce a reason to be shown in the tooltip. */
     disabledReason?: string | null | false
     noPadding?: boolean
-    size?: 'small' | 'medium' | 'large'
+    size?: 'tiny' | 'small' | 'medium' | 'large'
```

Comment:
I wonder if `xsmall` is better? Fits better with the tailwind-esque xs, sm, md, lg etc.

Reply:
Yep 👍 Ideally we'd move everything to the xs, sm, etc... legend